src/com/robibp/trening/Klinac.java

```diff
@@ -3,6 +3,9 @@
 public class Klinac {
     private String ime;
 
+    public Klinac( ) {
```

Comment:
Ovo je defaultni konstruktor koji ti java sama po sebi napravi ako ti ne napravis neki drugi. Tako da to mozes obrisat i opet ce radit

Reply:
Ako ti moj komentar nije jasan onda slobodno komentiraj na njega i pitaj, a ne ovako da ga ostavis da visi. Izbrisi taj konstruktor.